### ISSUE TITLE
Broken with twilio-ruby 5.30.0

### ISSUE THREAD

**Issue by ajsharp** [philnash/twiml_template#7]
twilio-ruby 5.30.0 does not define `Twilio::TwiML::Response`, which this library references below: 

https://github.com/philnash/twiml_template/blob/eba9966a5c27373208d520cd19fbe991b72e9a7f/lib/action_view/template/handlers/twiml.rb#L7

I believe this is a relic of an old version of twilio-ruby library.

**Comment by philnash** [comment]
That is correct, you should check out version 2 for support for `twilio-ruby` version 5 and up.

**Comment by philnash** [comment]
You might find you need to update your TwiML to work with the later version of `twilio-ruby`, check out [the release notes for a bit more detail](https://github.com/philnash/twiml_template/releases/tag/v2.0.0).

**Comment by ajsharp** [comment]
I'm using twiml_template 2.0. `Twilo::TwiML::Response` doesn't exist in the twilio-ruby 5 and up.

**Comment by philnash** [comment]
Ah, apologies. You're right, that was just something that escaped my update.

I think this needs to be changed to 

```ruby
 "self.output_buffer = ::TwimlTemplate::Response.new do |twiml|;" + 
```
Could you try to change this and test in your app to see if it helps? I am travelling today, so can't test easily.

Let me know if that works and I can make the updates once I'm settled.

Thanks!
